exp/performance.md

```diff
@@ -20,6 +21,16 @@ In addition, check your applications and make sure they use the resources in a f
 
 Generally, applications should use the smallest suitable instance type. However, if you see an overall bad performance when running the application, using a more powerful instance type usually helps (even though it reduces the container density). As an example, consider an application that runs on an Anbox Cloud deployment that does not have any GPUs installed. In this case, the rendering workload is put on the CPU instead of the GPU, and if the instance type of the application does not have a sufficient number of vCPU cores, the performance of the application is impacted. This can show, for example, in the virtual keyboard being really slow. By switching to a more powerful instance type, the container density is reduced, but the performance of each application container is increased.
 
+<a name="container-cpu-access"></a>
+## Container CPU access
+
+AMS has different modes to grant CPU access to a container. The `cpu.limit_mode` configuration option can be used to change the mode. The possible modes are:
+
+* `scheduler` - This will use the LXD [`limits.cpu.allowance`](https://linuxcontainers.org/lxd/docs/master/instances/#cpu-limits) <!-- wokeignore:rule=master --> configuration option to grant a container a CPU time budget via the Linux CFS scheduler. See [CFS Bandwidth Control](https://www.kernel.org/doc/html/latest/scheduler/sched-bwc.html) for more details.
+* `pinning` - This will use the LXD [`limits.cpu`](https://linuxcontainers.org/lxd/docs/master/instances/#cpu-limits) <!-- wokeignore:rule=master --> configuration option to pin a set of CPU cores to a container. LXD is responsible to allocate a specific number of cores to a container and load balance all running containers on all available cores.
```

Comment:
The container gets pinned to specific CPUs but which ones LXD is going to decide. If you look at the cpuset configuration of the container you will see specific cores being listed. LXD will load balance container across all available cores to find the best distribution.

Reply:
Yes, the explanation makes more sense. The term pinning used here introduces a little bit confusing to me. Since a **different** set of CPU cores will be allocated by LXD dynamically and pinned to **different** containers. So each CPU core may serve one or N containers depending on the number of CPU cores a host has,  also the number of containers run simentensoly.
I am fine to leave it as is.